bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -292,7 +292,7 @@ impl From<Time> for LockTime {
 impl PartialOrd for LockTime {
     #[inline]
     fn partial_cmp(&self, other: &LockTime) -> Option<Ordering> {
-        use LockTime::*;
+        use LockTime::{Blocks, Seconds};
```

Comment:
These ones are ok to leave in, they are enum variants.

Reply:
Thanks, I'll keep working on this and mark the PR as ready for review when done.

Reply:
I was hoping this was the answer, there are a lot of these.

Reply:
Should we update `CONTRIBUTING.md` to reflect this enum variant policy in https://github.com/rust-bitcoin/rust-bitcoin/blob/817e54fb83e27a0705e60cbdb520b1f26792e6d3/CONTRIBUTING.md?plain=1#L218 ?

Reply:
I think that's a good idea, add a policy that wildcard imports should not be used except in the three cases stated in #2875

Reply:
Done in: https://github.com/rust-bitcoin/rust-bitcoin/pull/2894